pkg/scheduler/framework/plugins/dynamicresources/dynamicresources_test.go

```diff
@@ -183,6 +183,8 @@ var (
 			ResourceClaims(resourceapi.ResourceClaimSchedulingStatus{Name: resourceName},
 			resourceapi.ResourceClaimSchedulingStatus{Name: resourceName2}).
 		Obj()
+	resourceSlice        = st.MakeResourceSlice(nodeName, driver).Device("instance-1", nil).Obj()
+	resourceSliceUpdated = st.FromResourceSlice(resourceSlice).Generation(1).Obj()
```

Comment:
I know the current implementation for `isSchedulableAfterResourceSliceChange` does not look into the specifics of what has been updated in the resource slice. That said, instead of just incrementing the generation, it might be better to write test cases based on actual resource slice update events (e.g., [when a device’s status changes to healthy](https://github.com/kubernetes/kubernetes/blob/06c6b9c0697b360ca66ad55df7bd16bb6caf4329/pkg/scheduler/framework/plugins/dynamicresources/dynamicresources_test.go#L105)). We could document it by implementing tests that take actual events into account.

Reply:
yes, this make sense